src/rules/utils.ts

```diff
@@ -633,6 +633,25 @@ export const isHook = (
   node.callee.type === AST_NODE_TYPES.Identifier &&
   HookName.hasOwnProperty(node.callee.name);
 
+export const getTestCallExpressionsFromDeclaredVariables = (
+  declaredVaiables: TSESLint.Scope.Variable[],
+): Array<JestFunctionCallExpression<TestCaseName>> => {
+  return declaredVaiables.reduce<
+    Array<JestFunctionCallExpression<TestCaseName>>
+  >((acc, { references }) => {
+    const callExpressions = references
```

Comment:
You can remove this intermediate variable by just calling `accc.concat(...)`.

That'll also let you short hand the arrow function, and avoid the extra overhead from copying both arrays.